fabric-resource-conditions-api-v1/src/main/java/net/fabricmc/fabric/impl/resource/conditions/ResourceConditionsImpl.java

```diff
@@ -204,4 +209,41 @@ public static boolean tagsPopulatedMatch(JsonObject object, RegistryKey<? extend
 
 		return true;
 	}
+
+	public static ConditionJsonProvider featureEnabled(Identifier id, final FeatureFlag feature) {
```

Comment:
You can simplify the id retrieval code as follows:
```java
			// Uses parameter: FeatureFlag... flags
			FeatureManager manager = FeatureFlags.FEATURE_MANAGER;
			FeatureSet set = manager.featureSetOf(flags);
			List<Identifier> identifiers = manager.toId(set).stream().sorted().toList();
```
This checks for the feature universe, and doesn't require the accessor mixin.